indico/modules/events/timetable/views/__init__.py

```diff
@@ -16,21 +16,26 @@
 
 from __future__ import unicode_literals
 
+import posixpath
+from itertools import groupby
 from operator import attrgetter
 
 from flask import render_template, request, session
 from pytz import timezone
 from sqlalchemy.orm import joinedload
 
-from indico.modules.events.layout import layout_settings
+from .weeks import inject_week_timetable
```

Comment:
OK, let's discuss this IRL or on IRC (`IR?`), it's pretty important to define a good practice.